Assets/Scripts/Enemy/Enemy.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+class Enemy : MonoBehaviour
+{
+    [Header("Stats")]
+    [SerializeField] private float _health;
+    [SerializeField] private float _damage;
+    //  This member will later influence the speed at which the enemy moves using the NavMeshAgent.
+    [SerializeField] private float _speed;
+    private bool _isAlive;
+
+    [Header("Movement")]
+    private StateMachine _stateHandler;
+
+    /*
+    [Header("Animation")]
+    Uncomment after integrating animation for enemy.
+    private EnemyAnimationHandler _animationHandler;
+    */
+
+    public float Health

```

Comment:
these should be auto properties https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties